src/settings.js

```diff
@@ -612,6 +614,9 @@ var DISABLE_EXCEPTION_CATCHING = 1;
 // [compile+link] - affects user code at compile and system libraries at link
 var EXCEPTION_CATCHING_WHITELIST = [];
 
+// New WebAssembly exception handling (experimental)
+var EXCEPTION_HANDLING = 0;
```

Comment:
Move to settings_internal.js?

Reply:
Done

Reply:
I tried that, but if I move this to settings_internal.js and if I do something like `em++ -fwasm-exceptions test.cpp`, we have an error:
```
$ em++ -fwasm-exceptions test.cpp
shared:ERROR: EXCEPTION_HANDLING is an internal setting and cannot be set from command line
```

Reply:
Moved to settings_internal.js successfully.